pandas/core/reshape/merge.py

```diff
@@ -1583,7 +1578,6 @@ def __init__(
         right_index: bool = False,
         axis: int = 1,
         suffixes: Suffixes = ("_x", "_y"),
-        copy: bool = True,
```

Comment:
I _think_ this should be retained since in the base class `_MergeOperation.__init__`.

Reply:
Reverted these changes